frontend/src/global_styles/content/_activity_list.sass

```diff
@@ -23,12 +23,24 @@
       @include spot-body-small
       color: $spot-color-basic-gray-2
 
+      &-comment
+        @include spot-body-small
+        list-style-type: disc
+        padding: $spot-spacing-0_5
+        padding-left: $spot-spacing-2
+        margin: 0
+
+      &-detail
+        &:not(:last-child)
+          margin-bottom: $spot-spacing-0_25
+
```

Comment:
See my above comment: if `op-activity-list--item-details` and `op-activity-list--item-detail` are reused, these two classes are not needed and we reduce duplication and future maintenance efforts.